docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/listing/list_item_macros.twig

```diff
@@ -332,15 +330,14 @@
         {% if media_type == "Video" or media_type == "Infographic" %}
            ({{ media_type }})
         {% elseif media_type == "File" %}
-          <span class="filetype {{file_type}}">
+          <span class="usa-collection__filetype">
             {% if file_type != "unknown" %}
-              {# Note, the spanish should have the word order reversed... #}
-              <span class="usa-sr-only">{{ file_type }} {% trans %}file{% endtrans %}</span>
+              [{{file_type}}
             {% else %}
-              {{ file_ext }}<span class="usa-sr-only"> {% trans %}file{% endtrans %}</span>
+              [{{ file_ext }}
             {% endif %}
+            , {{ file_size }}]
           </span>
```

Comment:
And one more https://github.com/NCIOCPL/cgov-digital-platform/pull/4903#discussion_r2325310192